docs/support-frontend-switches.md

```diff
@@ -2,10 +2,11 @@
 
 This app has a switchboard (at /switches) for enabling/disabling features on support.theguardian.com (support-frontend).
 
-The switches live in the support-admin-console S3 bucket, in `switches_v2.json`.
+The switches live in the support-admin-console S3 bucket, in `switches_v2.json`. Please donot copy `switches_v2.json` between Code, Dev, Prod because their settings may differ.
 
 The switchboard displays whatever switches are in the json file, and the support-admin-console [model](app/models/SupportFrontendSwitches.scala) does not have knowledge of individual switches.
 
-To add a new switch:
-1. Manually add it to the `switches_v2.json` json file
+To add/remove a new switch:
+
+1. Manually add/remove it to the `switches_v2.json` json file
```

Comment:
Would it be good here to give the name of the S3 bucket?

Reply:
Isn't it pretty clear higher up when it says -> 'The switches live in the support-admin-console S3 bucket, in `switches_v2.json`. '

Reply:
Ah yes, I hadn’t noticed that!